dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -328,6 +328,8 @@ export const ArticleMeta = ({
 							<Dateline
 								primaryDateline={primaryDateline}
 								secondaryDateline={secondaryDateline}
+								palette={palette}
+								format={format}
```

Comment:
Makes sense, if I understand correctly, you're suggesting, anywhere we need the palette, instead of passing palette as a component prop, we can just pass format. and always construct the palette within the component. 

I will make a change here as I think it'll make the dateline api cleaner.

Reply:
I know you said it's not necessary for this PR but I made the changes for `Dateline` and removed the palette from the Prop. Although I noticed in most of the other components the palette is being passed directly. If you think this change is making the code inconsistent, I'm happy to revert it

Reply:
Nice. I think it's fine to have some places use one pattern and others not. It's not a firm rule and we may well end up adapting more later on anyway.